pkg/controllers/node/emptiness.go

```diff
@@ -54,6 +56,11 @@ func (r *Emptiness) Reconcile(ctx context.Context, provisioner *v1alpha5.Provisi
 		return reconcile.Result{}, err
 	}
 
+	// node is empty, but it is in-use per the last scheduling round so we don't consider it empty
+	if r.cluster.IsNodeNominated(n.Name) {
+		return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
```

Comment:
Do we need to requeue? The reconcile loop should be re-triggered by the node's status updates or any related pods on this node.

Reply:
It will eventually retrigger, so it's probably fine to not re-queue.